src/createServer.js

```diff
@@ -1,9 +1,28 @@
 /* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  const server = http.createServer((req, res) => {
+    const [pathname, params] = req.url.split('?');
```

Comment:
Consider checking if `req.url` is defined before calling `split('?')` to avoid potential errors if `req.url` is undefined.